src/ajax.ts

```diff
@@ -37,7 +52,31 @@ function getConnector() {
  * @returns {Promise<any>}
  */
 export function send(url: string, data : any = {}, method = 'get', expectedDataType = 'json'): Promise<any> {
-  return getConnector().then((c) => c.send(url, data, method, expectedDataType));
+  if (method === 'get' || method === 'head') {
+    data = encodeParams(data); //encode in url
+    if (data) {
+      url += (/\?/.test(url) ? '&' : '?') + data;
+    }
+    data = null;
+  }
+  const options :any = {
```

Comment:
Is the ":any" required? I thought in the TS config there was a flag that was set so this wasn't required? Or should we be including it going forward and the flag is just stop gap?